pkg/oauth/client.go

```diff
@@ -70,6 +70,11 @@ type Client struct {
 
 	// XXX omitempty does not work for time.Time, thus the interface{} type
 	SynchronizedAt interface{} `json:"synchronized_at,omitempty"` // Date of the last synchronization, updated by /settings/synchronized
+
+	OnboardingSecret      string `json:"onboarding_secret,omitempty"`
+	OnboardingApp         string `json:"onboarding_app,omitempty"`
+	OnboardingPermissions string `json:"onboarding_permissions,omitempty"`
+	OnboardingState       string `json:"onboarding_state,omitempty"`
```

Comment:
How these fields will be filled? We need a CLI command to do that, no?

Reply:
Note: it can be done in another PR if you don't want to work too long on this one